modules/claims_api/app/swagger/claims_api/v2/production/swagger.json

```diff
@@ -7097,7 +7097,7 @@
                       "status": "422",
                       "detail": "Could not retrieve Power of Attorney due to multiple representatives with code: A1Q",
                       "source": {
-                        "pointer": "/modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney_controller.rb:85:in `representative'"
```

Comment:
I would vote to remove this. I also think we have a ticket to remove pointer fields like this from any 400 & 500 response. I don't like pointing to the line of code and it's not a requirement. 

I guess it will get updated every time we run RSWAG though.. but we could ignore it.